xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/MenuTranslations.xml

```diff
@@ -72,7 +72,30 @@ menu.uix.extensionPoint.value.panels.leftPanels=Inside a Left Panel
 menu.uix.content.label=UI Extension Content
 menu.uix.content.hint=If you can see this then it probably means you have JavaScript disabled. You should select the same value as for the menu display location.
 menu.uix.scope.label=Menu Visibility Scope
-menu.uix.scope.hint=Specifies in which context the menu is visible.</content>
+menu.uix.scope.hint=Specifies in which context the menu is visible.
+menu.description=This is a simple application that helps you create navigation menus to be displayed either horizontally as a top bar after the page header or vertically in a side panel.
+rendering.macro.menu.description=Displays a menu created using simple wiki syntax (nested lists and links).
+menu.macro.type.description=The optional menu type determines the menu appearance and behaviour. The supported values are: horizontal (default value) and vertical.
```

Comment:
This is not the right key. You used the right key below ``rendering.macro.menu.parameter.type.description``.